Cargo.lock

```diff
@@ -1294,6 +1329,32 @@ dependencies = [
  "group",
 ]
 
+[[package]]
+name = "parity-scale-codec"
```

Comment:
`parity-scale-codec` is Apache 2.0-only (not MIT) so needs an entry in `contrib/debian/copyright`.